### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by photous** [zealdocs/zeal#497]
when running zeal in the terminal it says:
```
 Segmentation fault (core dumped)
```
I have tried to `apt-get purge zeal` then install it again but it did not solve my problem.

**Comment by trollixx** [comment]
Try removing ~/.config/Zeal and ~/.local/share/Zeal.

**Comment by photous** [comment]
Thank you , It's working now :)
Do the docsets cause the issue?

**Comment by trollixx** [comment]
Yeah, apparently corrupted docset can cause a crash.

**Comment by photous** [comment]
How can I know which docset is corrupted?

**Comment by trollixx** [comment]
At the moment only by adding/removing docset folders one by one.

**Comment by photous** [comment]
Thank you very much, I have successfully found the problem. It was an unfinished docset.


Example of a full docset's `Contents` directory:
````
ls Ruby_on_Rails_4.docset/Contents/Resources/
docSet.dsidx      docSet.dsidx-wal  docSet.toc         Documents  Nodes.xml
docSet.dsidx-shm  docSet.mom        docSet.tokencache  LICENSE    Tokens.xml
```
The unfinished docset's `Contents` directory looks like this:
```
ls Twisted.docset/Contents/Resources/
docSet.dsidx  LICENSE
````

I hope this will give a clear view of the issue and makes it easy to fix  :+1:

**Comment by trollixx** [comment]
Thank you for the details. Hopefully fixed in f24039251c4d2e9e0d89ce19beb5a7763f26819b. Marking as a duplicate of #489, since it seems to be the same problem.
